.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,41 @@
+name: Generate SBOM with Kubernetes BOM
+
+on:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
```

Comment:
GitHub recommends setting minimum token permissions for the `GITHUB_TOKEN`. Although unlikely, there is a chance that when a new job is added to this workflow, its job permissions could be left undefined because of human error. We can prevent that by setting permissions definitions as read-only at the [top level](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions).